### DISCUSSION TITLE
What is JSON and its common operations

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#341] [Q&A] [answered]
In short

**Comment by d4tdev** [comment, marked as answer]
`JavaScript Object Notation (JSON)` is a standard text-based format for representing structured data based on JavaScript object syntax. It is commonly used for transmitting data in web applications (e.g., sending some data from the server to the client, so it can be displayed on a web page, or vice versa).

**Comment by Preetiraj3697** [comment]
JSON is an acronym for JavaScript Object Notation, is an open standard format, which is lightweight and text-based, designed explicitly for human-readable data interchange. It is a language-independent data format. It supports almost every kind of language, framework, and library.
